### ISSUE TITLE
Full battery icon looks much larger than regular battery icon, please make it smaller

### ISSUE THREAD

**Issue by nstommel** [numixproject/numix-icon-theme-circle#3295]
The 'full battery' icon looks much larger than the regular battery icon in Ubuntu, please make it smaller and more appealing. Unlike the regular charging/discharging icon, it hits the top and bottom of the global menu bar. This is a small but annoying issue that has been present for months:

![screenshot from 2016-07-14 20-51-16](https://cloud.githubusercontent.com/assets/15708310/16860361/872a21d2-4a05-11e6-83d3-65c7a6f7587f.png)

(and no, I didn't crop the top screenshot larger on purpose, sorry about that. But you get the point, it doesn't flow with the clean interface)

![screenshot from 2016-07-14 20-52-29](https://cloud.githubusercontent.com/assets/15708310/16860366/8af3afb8-4a05-11e6-9a73-283cb7984e21.png)

**Comment by palob** [comment]
Hi, both of your recently reported issues are actually issues for the [base theme](https://github.com/numixproject/numix-icon-theme).

About this issue, I can't reproduce it so it's gotta be only apparent with Unity.
I wonder why some of the icons look slighty blurry in your screenshot.
Did you resize the screenshots or do you use any non-integer HiDPI scaling?

**Comment by nstommel** [comment]
No, it's just blurry because I had to crop very small sections of full
1080p screenshots. It would be nice to have the cut-off blue circle with
arrow icons changed to something more aesthetically appealing. And yes,
it's standard Ubuntu 16.04 LTS with Unity.
Thanks for your consideration!

-Nick Stommel

**Comment by wa4557** [comment]
Are you using font scaling, different to 1? If yes then there's nothing we can do about that

**Comment by nstommel** [comment]
I run Ubuntu on a 13.3 inch laptop with a 1440p screen, so yes OF COURSE I use font scaling. Otherwise I wouldn't be able to see anything I was doing. Why do you say 'there's nothing we can do about that' when thousands of users probably have a similar configuration? I would appreciate if you could at least look into why the icon is so much larger than it should be. I know people love to bash Unity but really, Ubuntu as a distribution has been using it for such a long time that I think it's reasonable to at least look into the issue.

**Comment by wa4557** [comment]
@dirtydancing care to explain?

In short: it's virtually impossible to satisfy all different desktop environments. Fixing one breaks another, unfortunately. As a consequence using Ubuntu with font scaling different to one does not work. Fixing this would break all pure Gnome distros, I think. That's a pity, but what I meant with nothing we can do. 

You can fix this on your own though relatively easy by deleting a few icons from your icon folders (Right now I don't know which exactly but I will come back to that)

**Comment by palob** [comment]
If I were to own such screen I'd use a straight x2 HiDPI scaling.

Things might be a bit big then just like on a 720p one.

**Comment by dirtydancing** [comment]
Please cf. the big battery status icon issue over at https://github.com/numixproject/numix-icon-theme/issues/107. For a "workaround" for display scaling with Ubuntu/Unity https://github.com/numixproject/numix-icon-theme/issues/107#issuecomment-65576033. Cf. also this comment https://github.com/numixproject/numix-icon-theme/issues/107#issuecomment-59016997. Hope this is of help.

**Comment by nstommel** [comment]
@wa4557 Thanks, I guess that's just the way things are.

@palob I've found that many common applications still aren't compliant with the HiDPI display scaling used in Ubuntu, namely the gnome/gtk stuff (I'm not quite sure which really), so that means I have to settle with some blurriness and adjust the resolution from 1440p to 1080p on my 13.3inch screen if I want to see what I'm doing.

At least with stock Ubuntu there IS the option for fractional (not integer) display scaling. I've tried running other distros like Linux Mint (Ubuntu based and overall lovely on a large display) which unfortunately does NOT support fractional display scaling (only x1 and x2), which means things are either way too large or way too small...so it's an unusable mess on a laptop. (See [https://github.com/linuxmint/Cinnamon/issues/3455](url) )

Heck, even ol' Microsoft hasn't quite figured out HiDPI scaling. On 175% (Ideal for a 13.3'' 1440p display, as 150% is too small and 200% is too large) the Windows SYSTEM icons look downright awful. It's almost like 'seriously, did they test that?'. 

So, in short, display scaling for HiDPI displays kinda sucks. (I'm not blaming the Numix team here, I'm more just frustrated that the industry as a whole seems to just ignore the issue).

**Comment by nstommel** [comment]
@dirtydancing Whoah, I didn't realize this was such a noted issue. Just scrolling through all the comments is like 'huh? the system uses which icon?' Thanks for the reply, I will try the workaround you suggested.

**Comment by palob** [comment]
@nstommel For real? You've got a 1440p screen to run it with 1080p?

**Comment by nstommel** [comment]
@palob In all honesty, it's perfectly fine given how small the screen is. Now, if it were  a 17.x'' or ideally a 21.x'' display, I would of course use the full 1440p resolution. However, display scaling on a 13.3'' 1440p display isn't the greatest. Many applications are far more readable if the resolution is decreased to 1080p. I remember running Windows XP and Windows 7 on old laptops with 13.x'' screens and anything above 1366x768 was nigh on unreadable. So...this isn't all that bad.

**Comment by palob** [comment]
This is why I suggest display scaling by a factor of exactly 2. This will look like a 1280x720 screen with double sharpness.

**Comment by nstommel** [comment]
@palob Just tried it, everything looked huge but some things were crazy tiny. Opened nautilus, everything was freaking huge. Opened NetBeans, couldn't see what I was writing. Opened VLC, couldn't see the icons. So...even though things were sharper, it wasn't really more...usable. 1080p with 1.25 scaling hits the spot, so I keep it there. 
I'll give you the sharpness argument though, whenever I switch back to my Windows 10 partition, it's like whoooooah everything is so SHARP now. My current config on Ubuntu is alright though, when it comes down to it Full HD is fine with me just for consistency's sake.

**Comment by nstommel** [comment]
@dirtydancing YES, IT WORKS! No more ugly big battery with display scaling :smile: 

Here's a screenshot from my Ubuntu 16.04 desktop at 1080p with 1.25x display scaling. You can see that the battery indicator at full charge is now correctly sized! Thanks so much, this was really bugging me!

![screenshot from 2016-07-19 20-00-49](https://cloud.githubusercontent.com/assets/15708310/16970665/c2631706-4deb-11e6-83d6-5d402fc61c5d.png)

And here's the solution you suggested:
(I had to add the three directories 16x16  22x22  24x24  and file index.theme
under /usr/share/icons/No-Big-Battery/ and the 'status' subdirectories but it worked like a charm :smile: )

> Ok, these new developments prompted me to create a test/workaround for the "big battery" issue for myself. It has been asked in this thread before if there can be a "beginner-friendly" approach to this matter. I am sharing my approach here, so you can judge for yourself. Anyways, this should only take a couple of minutes, no root access is needed for this, and Numix-Circle/Numix icon themes are left untouched.

>This might be especially useful for users of Xubuntu 14.04 LTS or elementary OS. Again, all of this is a work in progress, but of course feedback from s.o. running these (or other) distributions on bare metal (i.e. not in a virtual machine) would be most welcome and helpful here.

>Here we go:

>1) Go to the hidden folder ~/.icons in your home directory. If you do not have such a folder, create one.

>2) Create a new folder in your ~/.icons directory, named No-Big-Battery.

>3) In ~/.icons/No-Big-Battery, create a new file and name it index.theme.

>4) Open the new index.theme file and copy & paste the following text into it. This is basically a stripped down index.theme of Numix icon theme. Of course, feel free to create your own index.theme if you feel like it.

```
[Icon Theme]
Name=No-Big-Battery

Comment=Test icon theme for a normal battery icon size in the panel

Inherits=Numix-Circle,Numix,gnome,hicolor

Example=folder

Directories=16x16/status,22x22/status,24x24/status

[16x16/status]
Size=16
Context=Status
Type=Fixed

[22x22/status]
Size=22
Context=Status
Type=Fixed

[24x24/status]
Size=24
Context=Status
Type=Fixed

```
>5) In ~/.icons/No-Big-Battery, create the new folders 16x16, 22x22 and 24x24. After that, in each of these folders, create the folder status.

>6) Copy the battery icons (for the time being, all the icons starting with "battery..."), from the three respective Numix status folders into your three new status folders.

>7) Select "No-Big-Battery" as your icon theme and see what happens. You might have to reboot your machine.

**Comment by dirtydancing** [comment]
@nstommel Great that this worked for you, and thanks for your feedback! :+1: 

There are different ways to work around this issue when using display scaling with Ubuntu/Unity, and apparently you opted to add an "extra" icon theme. The advantage of this approach (rather than changes within Numix icon theme) is that this fix will not be reset by Numix icon themes updates, e.g. via PPA, as such updates will not affect your "extra" icon theme.
